Copy edit FAQs (#767)
timescale/docs: timescaledb/overview/faq/faq-products.md

* If support for geospatial data is desirable. Data stored in TimescaleDB can leverage PostGIS's
geometric datatypes, indexes, and queries.

> * If you want greater optionality when it comes to using third-party tools. TimescaleDB supports

Reviewer: ```suggestion
* If you want more options when it comes to using third-party tools. TimescaleDB supports
```

> anything that speaks SQL (in other words, the entire PostgreSQL ecosystem).

Reviewer: ```suggestion
anything that speaks SQL, including the entire PostgreSQL ecosystem.
```

> * If you already use and like PostgreSQL, and don't want to "give it up" and move to a

Reviewer: ```suggestion
* If you already use and like PostgreSQL, and don't want to give it up and move to a
```

For this scenario, in-memory or column-oriented databases are designed for
key-value storage with fast lookup and a relational database may not be ideal.
However, these systems clearly do not scale to large data volumes and cannot

> perform well for more complex queries (where relational databases
> like TimescaleDB and PostgreSQL are better suited).

Reviewer: ```suggestion
perform well for more complex queries, where relational databases
like TimescaleDB and PostgreSQL are better suited.
```
